app/src/main/java/com/babylon/wallet/android/presentation/transfer/assets/AssetsTabs.kt

```diff
@@ -27,7 +27,7 @@ fun AssetsTabs(
     val tabIndex = remember(selectedTab) {
         ResourceTab.values().indexOf(selectedTab)
     }
-    ScrollableTabRow(
+    TabRow(
```

Comment:
why did you change it to be TabRow? I believe this ScrollableTabRow were used for a reason here, can't recall exactly, maybe it was that tab text was in 2 rows on some Samsung devices, something like this

Reply:
In bigger screens, with bigger width, I just realised that the scrollable tab row aligned everything to the left. It was impossible to show such tabs with a scrollable tab row. It is now fixed and centered correctly with a simple tab row.

Reply:
Guess it was for cases like this, but not sure if we can handle this any better.
<image src="https://github.com/radixdlt/babylon-wallet-android/assets/118203440/111568f0-1abd-49ba-a0cb-2ec169ce6a0a" width="30%">

Reply:
Oh do you see it like this now? What is the phone are you using?

Reply:
I see the same with my Galaxy

Reply:
It looked like this
![scrollable](https://github.com/radixdlt/babylon-wallet-android/assets/125959264/80e6083f-6ac1-46ad-b6b7-441f4d5afe0a)

Reply:
By default this is its wrap content behaviour. Added red as background to for you to see.
![red](https://github.com/radixdlt/babylon-wallet-android/assets/125959264/baabc0cf-54fc-43b8-b78b-747be501e3ec)

Reply:
@micbakos-rdx I use samsung s22 ultra, but I think it is related to system font size setting

Reply:
Scrollable tab row is implemented inside with 

```
SubcomposeLayout(
            Modifier
                .fillMaxWidth().  <------
                .wrapContentSize(align = Alignment.CenterStart)
                .horizontalScroll(scrollState)
                .selectableGroup()
                .clipToBounds()
        )
```

Reply:
Removed inner padding from tabs and added this padding to the whole tab row. Not a perfect solution. For such design I would consider ditching TabRow altogether and creating our own since both Scrollable and simple tab rows, do not allow for wrap content.